From e3431259c2441d2359823c490d9b32f825012565 Mon Sep 17 00:00:00 2001
From: Nadav Haklai <nadavh@marvell.com>
Date: Thu, 22 Oct 2015 13:28:44 +0300
Subject: [PATCH 0449/1240] armada3700: Update default bootargs for config
 palladium

- Update correct earlycon driver (ar3700_uart)
- Add temp WA for Linux SWIOTLB bug

Change-Id: Ia2d56e74150d40aac3b197bcfe41bd01010b0cf1
Signed-off-by: Nadav Haklai <nadavh@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/24223
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Hanna Hawa <hannah@marvell.com>
---
 include/configs/armadalp.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/configs/armadalp.h b/include/configs/armadalp.h
index f28487f..2b70a24 100644
--- a/include/configs/armadalp.h
+++ b/include/configs/armadalp.h
@@ -110,7 +110,7 @@ extern char __end_of_spl[];
 #endif
 
 #undef CONFIG_BOOTARGS
-#define CONFIG_BOOTARGS			"console=ttyS0,300 earlycon=arlp_uart,0xd0012000"
+#define CONFIG_BOOTARGS			"console=ttyS0,300 earlycon=ar3700_uart,0xd0012000 swiotlb=4096"
 
 #undef  CONFIG_EXTRA_ENV_SETTINGS
 #define CONFIG_EXTRA_ENV_SETTINGS	"kernel_addr=0x107ffc0\0"	\
-- 
1.9.1

